arangod/IResearch/IResearchDataStore.h

```diff
@@ -105,6 +107,11 @@ class IResearchDataStore {
       // for now we require that each index has its own snapshot
       TRI_ASSERT(_snapshot);
     }
+
+    [[nodiscard]] TRI_voc_tick_t tick() const noexcept {
+      return _snapshot->tick();
+    }
+
```

Comment:
_snapshot is public here, is it really necessary?

```suggestion
```

Reply:
yes, I like it more that way.